### ISSUE TITLE
Initial test

### ISSUE THREAD

**Issue by paaljoachim** [pojome/elementor#18]
Hey

Man I was impressed by your launch page and included demos. So much so that I wanted to take your plugin for a test run and share my initial test along the way.

I have just installed Elemontor. Having clear "arrows" where to go next is helpful.
Some plugins add a settings like so: Activate I Edit I Delete I Settings to make is fully clear where the settings for the plugin is located. (For those people who have too many plugins in the left sidebar and just can not locate it..:)

Nice Elementor icon!
Elementor settings page is the first page the user likely sees. Use it to promote the plugin features. Add some documentation a 5 minute video tutorial showing where to go from here OR/And some initial instructions that can be hidden away. Remember the user will not even have tested the plugin yet so it is good to give some guidance on what is next. As right now it is unclear on how to proceed.

I went to one of my posts.
A very huge red button seems almost like it is flashing in my face sits right below the Permalink.
I am looking at making the button smaller.
Such as

`
#elementor-switch-mode button.elementor-switch-mode-button {
    padding: 0 30px;
    height: 40px;
    /* line-height: 50px; */
    width: 150px;
}
`

Pushing the text perhaps within 5 px of the left edge of the button.
Make the button a little smaller please.

Pressing to turn on the Elementor shows for a second the backend content screen then jumps to the frontend. Wow that was quick!

I first began clicking some of the Elements and nothing happened. Tried clicking the site title, top menu some images..nothing happened. Scrolled down and noticed that I could click within the content... Well I was editing the post content so that seems natural.

The content was seen within a tinymce Visual editing box very similar to the backend tinymce. I noticed the arrow pointing left and clicked it. It removed the Elementor section on the left. I moved the cursor on the edge between the two areas thinking that I could extend the Elementor section but that I could not do. That meant I saw this cramped area on the left showing the post content.
I clicked inside the post content right side box again thinking I could edit the text directly on the page but that did not work. I have to edit the content on the left.

** Editing content directly on the page in the right section along with inside the Elementor section to the left.

An inline toolbar showed up showing a pencil/edit - copy icon - x delete. This showed up every time I clicked inside the content. Not sure why right now....

Column-> Section-> duplicate and delete

I clicked column. Clicked classic whatever that means.

Clicked color and no change was seen in the content on the right. Then I noticed that there was an image saying some words. I clicked Image and changed it for something else and then deleted the image and the background color was seen. Perhaps start out with no image? So the background color is seen right away.

** Start with no image so using border color it is seen right away.

Border type, and then selecting a color. When I select a border color I want to see the effects right away. So default after having selected color could be to have 1 px locked so that I see right away the Width box right above.

** Selecting border color have a default border width show up in the border width box, so the border is seen right away.

I continued experimenting...

I dragged a column element into the content and dropped it. I dragged another as well. I have to column elements. Dragged an image element into it and found an image. I dragged a text block and dropped it below the image element in the same column. 
I thought I would be able to click the image and resize it but that did not work. I Tried resizing the text element that did not work. I can resize columns but it seems I can not resize other elements. Ahh there is a spacer element that is neat.

What I missed.
Dragging out the Elementor section making it bigger to have more space to edit whatever I have selected on the right side.
Editing the full page if possible - Title, logo, menus etc etc.... Hmm might be a bit much to ask for.

I clicked a structure section in the Edit Section and it showed a down arrow structure showing a grey box. I am guessing something will happen when I click the correct area.

Add New Section...

Very neat! Creating a column like structure area.

I am going to stop writing now and just continue experimenting...

The bottom line....well I might have readjusted some of the stuff I mentioned above. It is so very important to work on consistency. Select your structure seems similar to columns.
There was some confusion at various points. It is great to see the various widget/elements I have not tried out yet. Some of the hurdles kinda put me out but with anything new it just takes a while to get into it. Hopefully the enjoyment is bigger then the learning hurdles.

Ok just a few more things.
When to edit when to preview. Adding a toggle It would be nice with a in your face tooltip a play icon to preview it.
Button. Type - switch this to select color wheel thingie. Border radius. Border color. Other options look nice!

**Comment by KingYes** [comment]
Thanks. We will review it and take it into consideration. There are a lot more updates coming up soon, so please stay updated.

**Comment by paaljoachim** [comment]
Let me know when you have some additional updates so that I can create test number 2.

**Comment by arielk** [comment]
- In the last version we added the settings button on the plugins screen, thanks.

- In the next version we make the button smaller. thanks again.

- We working on new feature that you can the write live on the page, but it take some time. we known this uselessness

Anyway thanks for your feedback. it was very enlightening. 



Just remember that Elementor is a Page Builder but not a Theme builder, so we keep focus on in. So please keep updated we released a new version few times a week.

**Comment by benpines** [comment]
Hi Paal,
Wow, that's a lot of feedback :)
We really appreciate the thorough examination you did on our product.
Your feedback, however long, was very insightful and deep.
Here is the answers:
- In the final version we added a settings button in the plugins page, thanks for the idea.
- In the next version we have designed a smaller button "edit with Elementor", less obtrusive like you suggested. We will release it today.
- We are working on a feature to allow writing on the page display area, but this could take a bit of time. We know this is a needed feature.
- Remember that Elementor is a page builder, not a theme builder, so you cannot do changes -
- Did you use Elementor with Genesis? I'd love it if you could write about the compatibility of them.
beyond the content of the page. The header and footer, that's the theme domain. 
We will be releasing more updates in the next month, so keep updating.

We value feedback like yours at the highest importance, and this is why I have to ask you to make it shorter, so we can benefit the most from it. Please open a separate issue for each subject matter, so it doesn't get lost. Our programmers like to take it one issue at a time. I suggest this method:
1. The current situation:
2. The problem with this:
3. My suggestion for a fix:

Can't wait for test 2,
Ben

**Comment by paaljoachim** [comment]
Within a few weeks I will create a test 2 with a Genesis child theme. Then create separate issues for each matter that shows up.
